src/lsm/manifest_level.zig

```diff
@@ -119,17 +230,22 @@ pub fn ManifestLevelType(
         /// * Asserts that the table exists in the manifest.
         pub fn set_snapshot_max(level: *Self, snapshot: u64, table_ref: TableInfoReference) void {
             var table = table_ref.table_info;
-            const key_min = table.key_min;
-            const key_max = table.key_max;
```

Comment:
Good catch, this eliminates a stack copy I believe?

Reply:
Yes! 👍🏼